downloads/tests/settings/19.py

```diff
@@ -0,0 +1,97 @@
+import os
+from os.path import expanduser
+
+SECRET_KEY = "PLACE_SECRET_KEY_HERE"
+
+USE_TZ = True
+
+TIME_ZONE = "Africa/Johannesburg"
+
+DATABASES = {
+    'default': {
+        'ENGINE': 'django.db.backends.sqlite3',
+        'NAME': 'mydatabase'
+    }
+}
+
+INSTALLED_APPS = (
+    "downloads",
+    "test_without_migrations",
```

Comment:
See if you can get rid of this. I added it in another product but realistically we should be testing migrations.